File: queue/queue_implementation.js
class Queue{
    constructor(storage = []){
        this.storage = storage;
    }

    lookup(){
        return this.storage[0]
    }

    push(value){
        this.storage.push(value);
        return this.storage
    }

    shift(){
        this.storage.shift();
        return this.storage
    }

}

let obj = new Queue();

File: Elvis/queue/queue_implementation.py
class queue:
    def __init__(self, storage = []):
        self.storage = storage
    def lookup(self):
        return self.storage[0]
    def shift(self):
        del self.storage[0]
    def push(self, value):
        self.storage.append(value)
        return self.storage



